test/sets/user_contexts.rb

```diff
@@ -0,0 +1,19 @@
+module Contexts
+  module UserContexts
+
+    # Create three Questions
+    def create_user
```

Comment:
should follow the same naming conventions as before and user plural
so create_users and remove_users